be/src/exec/hash_joiner.cpp

```diff
@@ -573,7 +573,20 @@ Status HashJoiner::_create_runtime_bloom_filters(RuntimeState* state, int64_t li
         int expr_order = rf_desc->build_expr_order();
         ColumnPtr column = ht.get_key_columns()[expr_order];
         bool eq_null = _is_null_safes[expr_order];
-        _runtime_bloom_filter_build_params.emplace_back(pipeline::RuntimeBloomFilterBuildParam(eq_null, column));
+        MutableJoinRuntimeFilterPtr filter = nullptr;
+        auto multi_partitioned = rf_desc->layout().pipeline_level_multi_partitioned();
```

Comment:
should we consider compatibility issues? during upgrading, TRuntimeFilterDescription generated by older FE won't contain layout field, seems we should check has_layout() first?

Reply:
It is safe in gray-scale upgradation.  two reasons:
1. Only when all the nodes of the cluster have been updated, then we can turn on this feature.
2. Assuming in gray-scale upgradation, some BE(s) are old, some BE(s) are new, this feature is off.  the new ones and the old ones generate runtime filters of the same layouts(SINGLETON, GLOBAL_SHUFFLE_1L, GLOBAL_BUCKET_1L).

Reply:
Soundness of gray-scale upgradation has been testified